src/content/self-hosted/manage/buildkit-high-performance.mdx

```diff
@@ -0,0 +1,122 @@
+---
+title: Optimizing Build Performance
+description: Best Practices for Optimal Build Performance
```

Comment:
I'm not sure the title reflects the pain point the admin/developers have.  Maybe we can call it "Build Service Performance Guide"  and "Recommendations on how to speed up your builds"?

Reply:
I am going with "Configuring Buildkit for High-Performance". I don't want to mention "build" because this applies to deploy/test/destroy... I will make this clear at the beginning of the doc